src/tree/AppResourceTreeItem.ts

```diff
@@ -46,7 +45,7 @@ export class AppResourceTreeItem extends ResolvableTreeItemBase implements Group
         this.groupConfig = createGroupConfigFromResource(resource, root.id);
 
         this.contextValues.add(AppResourceTreeItem.contextValue);
-        ext.tagFS.fireSoon({ type: FileChangeType.Changed, item: this });
+        // ext.tagFS.fireSoon({ type: FileChangeType.Changed, item: this });
```

Comment:
Could you explain why you commented this out? Should it go back eventually?

Reply:
This is a v1 class that isn't used anymore. It's a lot of work to delete everything, will do in a follow up PR.

Reply:
When I start deleting files it just shows how much there is left to do. For example:
1. Toggle show all resources command
2. Group by tags needs updated so that it doesn't rely on the root tree item to get subscriptions
3. `revealTreeItem`

I'm sure there's more